projects/ngrx.io/content/guide/migration/v20.md

```diff
@@ -7,7 +7,11 @@ NgRx supports using the Angular CLI `ng update` command to update your dependenc
 To update your packages to the latest released version, run the command below.
 
 ```sh
-ng update @ngrx/store@20
+# NgRx Store related packages
+ng update @ngrx/store@20-beta.0
+
+# NgRx Signals package
+ng update @ngrx/signals@20-beta.0
```

Comment:
I tried this and it did not work, but `ng update @ngrx/signals@20.0.0-beta.0` did. Is this PR pre-emptive of a `20-beta.0` release or is this `.md` copy just a bit off?

Reply:
Thanks for the catch! This was pre-emptive to not forget about adding a version, but I forgot to add the correct full version.